src/Web/Sfa.Das.Sas.Core/Domain/Helpers/UrlEncoder.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Sfa.Das.Sas.Core.Domain.Helpers
+{
+    public class UrlEncoder : IUrlEncoder
+    {
+        public string EncodeTextForUri(string textToProcess)
+        {
+            var specialCharactersRemoved = Regex.Replace(textToProcess.ToLower(), @"['-]", string.Empty);
+            var splitBySpacesAndOtherChars = Regex.Split(specialCharactersRemoved, @"[\s(),.:;?!\\/]+");
```

Comment:
possible System.NullException error on line 10

Reply:
I'll change this to return string.Empty - spoke to Scott and he suggested that if the title is null we have a bigger problem

Reply:
I've updated the method to return string.Empty